config/controller/controller.yaml

```diff
@@ -31,8 +31,8 @@ spec:
       - args:
           - --cluster-name=CLUSTER_NAME
           - --role-arn=USER_ROLE_ARN
-          - --enable-leader-election
-          - --metrics-addr=:8443
+          - --leader-elect
```

Comment:
Is there a reason we need to change their names? I saw the code has been updated as well but this will break our deployment if we forget updating our deployment workflow. I would like to understand why we need to change flags' name.

Reply:
Yes, the manager flags are renamed in v3 https://book.kubebuilder.io/migration/legacy/manually_migration_guide_v2_v3#rename-the-manager-flags